README.md

```diff
@@ -1,5 +1,13 @@
-This repository contains a list of tests in the Salt Test Suite that are **not** executed. Skipping
-tests is bad and should only be used as a last resort.
+This repository contains a list of tests in the Salt Test Suite that are **not** executed. Remember: **Skipping tests is bad and should only be used as a last resort.**
 
-The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing
-groups do  not have t be listed explicitely.
\ No newline at end of file
+The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing groups do not have to be listed explicitely. The list is used by `salt-test`, a test launcher that skips the collection of files in the `[ignore]` sections and deselects all tests in the `[skip]` section.
+
+### `ignore` vs `skip`
+
+Rule of thumb: Use `skip` if you can, `ignore` if you must.
+Reason: `ignore` skips over a whole Python module (i.e. a file). It's needed when the import of the module causes errors, e.g. `ImportError`. We don't have all of Salt's Python dependencies packaged and need to exclude files that don't guard their imports.
+`skip` has finer granularity and is prefered for that reason.
+
+### Adding new entries to `skipped_tests.toml`
```

Comment:
Would it make sense to also add a small para on executing the tests? Right now, I don't know how to test them, though maybe that's what Yeray might add later?

Reply:
Maybe we could link to https://github.com/openSUSE/salt-test#readme

Reply:
Ah that'd be great, we already say that this is used by "`salt-test`"; instead of making it a code, we could make the salt-test link to the repo.

And I assume we'll later document in `salt-test` how to execute the container.